language/move-lang/src/parser/syntax.rs

```diff
@@ -1581,24 +1602,14 @@ fn parse_invariant<'input>(tokens: &mut Lexer<'input>) -> Result<SpecBlockMember
 }
 
 // Parse a specification function.
-//     SpecFunction = "define" <Generics> <Parameters> ":" <Type> "{" <Sequence> "}"
-//                  | "native" "define" <Generics> <Parameters> ":" <Type> ";"
+//     SpecFunction =
+//        "native"? "define" <Name> <OptionalTypeParameters> <Parameters> ":" <Type> "{" <Sequence> "}"
```

Comment:
Thanks for fixing the "Generics" reference here. The "\<Parameters\>" reference is not defined anywhere in one of these grammar comments. It's simple enough that I would just replace it with the comment that you have later in the function.  Merging the native variant doesn't look right to me, since it still ends with a semicolon instead of a brace-enclosed Sequence.

Reply:
Good catch. Done.